internal/action/bufpane.go

```diff
@@ -353,14 +351,13 @@ func (h *BufPane) OpenBuffer(b *buffer.Buffer) {
 	h.Buf = b
 	h.BWindow.SetBuffer(b)
 	h.Cursor = b.GetActiveCursor()
+	// Set the cursors IsOverwriteMode by default to false on open
+	h.Cursor.IsOverwriteMode = false
```

Comment:
What about other cursors?
I've tried using overwrite mode with multicursors with your PR. It doesn't work very well.

Reply:
Yep, that was the reason why I set it into draft short after publishing the PR.
I remembered that I forget about the multi-cursors once again...shame on me.